boards.txt

```diff
@@ -15,222 +15,77 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 
-adafruit_feather_m0.name=Adafruit Feather M0
-adafruit_feather_m0.vid.0=0x239A
-adafruit_feather_m0.pid.0=0x800B
```

Comment:
I'm not sure we should be deleting the feather_m0.... this code is useful to more people than MCCI. Instead we should just add our extra material.